setup.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2013 Hewlett-Packard Development Company, L.P.
```

Comment:
For tox. I can investigate how to do this in a nicer way that doesn't flood the top dir